src/AccessibilityInsights.SharedUx/Telemetry/TelemetryAction.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft. All rights reserved.
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This is an automatic change to add the byte order marker. Should I uncommit it?

Reply:
No. We want these